blosc/blosc.c

```diff
@@ -1731,10 +1747,11 @@ int blosc_getitem_unsafe(const void *src, int start, int nitems, void *dest) {
   return blosc_getitem_impl(src, start, nitems, dest, /*unsafe=*/1);
 }
 
-/* Decompress & unshuffle several blocks in a single thread */
-static void *t_blosc(void *ctxt)
+/* execute single compression/decompression job for a single thread_context */
+static void t_blosc_do_job(void *ctxt)
```

Comment:
Note that this code is a trivial refactoring — the `t_blosc_do_job` is simply cut-and-pasted out of the original `t_blosc` function.